machetli/pddl/files.py

```diff
@@ -11,18 +11,18 @@
 DIN = "  "  # double indentation
 
 
-def generate_initial_state(domain_filename: str, problem_filename: str) -> dict:
+def generate_initial_state(filename: str, filename: str) -> dict:
```

Comment:
This looks problematic. I'd stick with the old names here.

Reply:
Indeed, this is unintentional. Must have happened when I auto-renamed stuff and I didn't realize. Thanks for catching this!